github.go

```diff
@@ -54,37 +56,57 @@ func runGitHubHandler(c *gin.Context) {
 
 	importer.Token, err = importer.getToken(i)
 	if err != nil {
-		c.Error(err)
-		c.JSON(http.StatusBadRequest, fmt.Sprintf("Unable to get token. %+v", err))
+		ginutil.WriteProblemError(c, err, problem.Response{
+			Type: "prob/provider/github/getting-token-error",
+			Title: "Error getting token.",
+			Status: http.StatusBadRequest,
```

Comment:
Would declare this more as a "Bad Gateway" error, as the error occurred when acting as a gateway to the API via the `importer.getToken` method.

Reply:
Same comment for the other problems